etl/steps/data/garden/regions/2023-01-01/regions.yml

```diff
@@ -1847,3 +1847,221 @@
   end_year: 1902
   successors:
     - "ZAF"
+
+# WHO regios
+- code: WHO_AMR
+  name: "WHO Americas region"
+  region_type: "aggregate"
+  members:
+    - "ATG"
+    - "ARG"
+    - "BHS"
+    - "BRB"
+    - "BOL"
+    - "BRA"
+    - "CAN"
+    - "CHL"
+    - "COL"
+    - "CRI"
+    - "CUB"
+    - "DMA"
+    - "DOM"
+    - "ECU"
+    - "SLV"
+    - "GRD"
+    - "GTM"
+    - "HTI"
+    - "HND"
+    - "JAM"
+    - "MEX"
+    - "NIC"
+    - "PAN"
+    - "PRY"
+    - "PER"
+    - "KNA"
+    - "LCA"
+    - "SUR"
+    - "TTO"
+    - "USA"
+    - "URY"
+    - "VEN"
+
+- code: WHO_AFR
+  name: "WHO African region"
+  region_type: "aggregate"
+  members:
+    - "DZA"
+    - "AGO"
+    - "BEN"
+    - "BWA"
+    - "BFA"
+    - "BDI"
+    - "CMR"
+    - "CPV"
+    - "CAF"
+    - "TCD"
+    - "COM"
+    - "COG"
+    - "CIV"
+    - "COD"
+    - "GNQ"
+    - "ERI"
+    - "SWZ"
+    - "ETH"
+    - "GAB"
+    - "GMB"
+    - "GHA"
+    - "GIN"
+    - "GNB"
+    - "KEN"
+    - "LSO"
+    - "LBR"
+    - "MDG"
+    - "MWI"
+    - "MLI"
+    - "MRT"
+    - "MUS"
+    - "MOZ"
+    - "NAM"
+    - "NER"
+    - "NGA"
+    - "RWA"
+    - "STP"
+    - "SEN"
+    - "SYC"
+    - "SLE"
+    - "ZAF"
+    - "SSD"
+    - "TZA"
+    - "TGO"
+    - "UGA"
+    - "ZMB"
+    - "ZWE"
+- code: WHO_EMR
+  name: "WHO Eastern Mediterranean region"
+  region_type: "aggregate"
+  members:
+    - "AFG"
+    - "BHR"
+    - "DJI"
+    - "EGY"
+    - "IRN"
+    - "IRQ"
+    - "JOR"
+    - "KWT"
+    - "LBN"
+    - "LBY"
+    - "MAR"
+    - "OMN"
+    - "PAK"
+    - "QAT"
+    - "SAU"
+    - "SOM"
+    - "SDN"
+    - "SYR"
+    - "TUN"
+    - "ARE"
+    - "YEM"
+- code: WHO_EUR
+  name: "WHO European region"
+  region_type: "aggregate"
+  members:
+    - "ALB"
+    - "AND"
+    - "ARM"
+    - "AUT"
+    - "AZE"
+    - "BLR"
+    - "BEL"
+    - "BIH"
+    - "BGR"
+    - "HRV"
+    - "CYP"
+    - "CZE"
+    - "DNK"
+    - "EST"
+    - "FIN"
+    - "FRA"
+    - "GEO"
+    - "DEU"
+    - "GRC"
+    - "HUN"
+    - "ISL"
+    - "IRL"
+    - "ISR"
+    - "ITA"
+    - "KAZ"
+    - "KGZ"
+    - "LVA"
+    - "LTU"
+    - "LUX"
+    - "MLT"
+    - "MDA"
+    - "MCO"
+    - "MNE"
+    - "NLD"
+    - "MKD"
+    - "NOR"
+    - "POL"
+    - "PRT"
+    - "ROU"
+    - "RUS"
+    - "SMR"
+    - "SRB"
+    - "SVK"
+    - "SVN"
+    - "ESP"
+    - "SWE"
+    - "CHE"
+    - "TJK"
+    - "TUR"
+    - "TKM"
+    - "UKR"
+    - "GBR"
+    - "UZB"
+- code: WHO_SEAR
+  name: "WHO South-East Asia region"
+  region_type: "aggregate"
+  members:
+    - "BGD"
+    - "BTN"
+    - "PRK"
+    - "IND"
+    - "IDN"
+    - "MDV"
+    - "MMR"
+    - "NPL"
+    - "THA"
+    - "TLS"
+    - "LKA"
+- code: WHO_WPAC
+- name: "WHO Western Pacific region"
```

Comment:
Here there is an issue: I think you need to remove the extra "-" on the left of "name". Otherwise the previous item is stored as a region without a name, which causes other problems (I think that's why `nuclear_weapons_proliferation` step is failing [in buildkite](https://buildkite.com/our-world-in-data/etl-automated-staging-environment/builds/4049#018ee124-61e7-45c3-a6c7-9b859ff59369)).